plugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -10,16 +11,37 @@
     </bean>
     <service ref="connectionManager" interface="org.opennms.velocloud.connections.ConnectionManager"/>
 
+    <bean id="connectionManager" class="org.opennms.velocloud.connections.ConnectionManager">
```

Comment:
This is a duplicate of line 7 which causes issues

Reply:
Thanks, Ive been merging the new stuff from develop and I missed that part. I hope to do an update soon.